sql/dbschema.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE users (
+  userId INT PRIMARY KEY IDENTITY(1,1),
+  firstName VARCHAR(50) NOT NULL,
+  lastName VARCHAR(50) NOT NULL,
+  email VARCHAR(50) NOT NULL,
+  deleted INT DEFAULT 0 NOT NULL
+);
+
+CREATE TABLE tasks(
+  taskId INT PRIMARY KEY IDENTITY(1,1),
+  task VARCHAR(300),
```

Comment:
What does `task` mean in this context? Is this the task's `title` or `description`? Maybe having two fields - one for short title and another for long description is more appropriate.